RallyHereIntegration/Source/RallyHereIntegration/Private/RallyHereIntegrationModule.cpp

```diff
@@ -40,5 +43,4 @@ void FRallyHereIntegrationModule::ShutdownModule()
     }
 
 	FRH_PollControl::Uninitialize();
```

Comment:
Should the poll control also be uninitialized before?

Reply:
Preferably they would both be after, but the async one runs cancels on every still firing task, which ends up depending on the integration layer.  The actual async system doesnt, but the things it uses do.

The poll control is self contained and so shutting it down last (in case anything tries to unregister itself) is preferred.